libs/protocol-ui/src/stories/molecules/GovrnCta.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Box, Button } from '@chakra-ui/react';
+import { GovrnCta, GovrnCtaProps } from '../..';
+
+export default {
+  title: 'Components/Molecules/GovrnCta',
+  component: GovrnCta,
+};
+
+const Buttons = [
+  <Button variant="primary">Report Your First Contribution</Button>,
+  <Button variant="secondary">Learn More About Kevin Malone</Button>,
+  <Button variant="tertiary">Third Call to Action</Button>,
+];
+
+export const NoButtons = (args: GovrnCtaProps) => (
+  <Box backgroundColor="gray.50">
+    <Box padding={10}>
+      <GovrnCta {...sharedArgs} />
+    </Box>
+  </Box>
+);
+
+export const OneButton = (args: GovrnCtaProps) => (
+  <Box backgroundColor="gray.50">
+    <Box padding={10}>
+      <GovrnCta {...sharedArgs} children={Buttons.slice(0, 1)} />
+    </Box>
+  </Box>
+);
+
+export const TwoButtons = (args: GovrnCtaProps) => (
+  <Box backgroundColor="gray.50">
+    <Box padding={10}>
+      <GovrnCta {...sharedArgs} children={Buttons.slice(0, 2)} />
+    </Box>
+  </Box>
+);
+
+export const ThreeButtons = (args: GovrnCtaProps) => (
```

Comment:
These seem a bit excessive. We should probably only have one component and pass in the appropriate buttons where they are needed. This will give us more flexibility

Reply:
Oh these are just demos for the story. It takes in a child which is however many elements it needs.

Reply:
Actual component only takes in children - this shows how it looks with 0-3

Reply:
Ah okay cool